docs/src/example.md

```diff
@@ -0,0 +1,265 @@
+# Example
+
+In this example, we're going to step through a set of common operations we typically perform when converting a collection of individually fetched features into a simple set of training fetures (X, y) and predict/testing features (X̂, ŷ).
+
+Lets start by loading some packages we'll need.
+
+```@example full
+using AxisKeys, AxisSets, DataFrames, Dates, Impute, Random, TimeZones
+using AxisSets: Pattern, flatten, rekey
+```
+
+## Data
+
+```@setup full
+# Only gonna lookback 4 days on the same hour and our training data only covers 1 week
+train_input_times = DateTime(2021, 1, 1):Hour(1):DateTime(2021, 1, 7)
+train_output_times = train_input_times .+ Day(1)
+
+# Predict times are just the next day
+predict_input_times = DateTime(2021, 1, 7):Hour(1):DateTime(2021, 1, 8)
+predict_output_times = predict_input_times .+ Day(1)
+
+# We're gonna say that price, load and temp nodes are different non-overlapping ids
+node_ids = [:a, :b, :c, :d]
```

Comment:
This says `node_ids` but comment says `price`

Reply:
Yeah, changed it to `price_ids`